packages/cf-component-progress/src/Progress.js

```diff
@@ -1,6 +1,95 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Link from 'cf-component-link';
+import { createComponent } from 'cf-style-container';
+
+const styles = () => ({
+  position: 'relative'
+});
+
+const Bar = createComponent(
+  () => ({
+    display: 'block',
+    width: '100%',
+    appearance: 'none',
+
+    '&::-webkit-progress-bar': {
+      backgroundColor: '#ebebeb',
+      color: '#ebebeb'
+    },
+
+    '&::-webkit-progress-value': {
+      backgroundColor: '#2f7bbf',
+      color: '#2f7bbf',
+      transition: 'width 500ms ease'
+    },
+
+    /* Mozilla uses progress-bar as the value */
+    '&::-moz-progress-bar': {
+      backgroundColor: '#2f7bbf',
+      color: '#2f7bbf'
+    }
+  }),
+  'progress',
+  ['max', 'value']
+);
+
+const Items = createComponent(
+  () => ({
+    listStyle: 'none',
+    '&::after': {
+      content: '',
```

Comment:
This is super easy to miss. I've done this very recently where I caught this before shipping but I wonder if there's benefit in doing something like this automatically.